src/main/kotlin/org/rust/lang/core/types/Extensions.kt

```diff
@@ -15,11 +15,25 @@ import org.rust.lang.core.psi.*
 import org.rust.lang.core.psi.ext.*
 import org.rust.lang.core.resolve.ImplLookup
 import org.rust.lang.core.resolve.StdKnownItems
+import org.rust.lang.core.types.borrowck.BorrowCheckContext
+import org.rust.lang.core.types.borrowck.BorrowCheckResult
 import org.rust.lang.core.types.infer.*
-import org.rust.lang.core.types.ty.*
+import org.rust.lang.core.types.ty.Mutability
+import org.rust.lang.core.types.ty.Ty
+import org.rust.lang.core.types.ty.TyTypeParameter
+import org.rust.lang.core.types.ty.TyUnknown
 import org.rust.openapiext.recursionGuard
 
 
+private fun <T> RsInferenceContextOwner.createResult(value: T): Result<T> {
+    val projectTracker = project.rustStructureModificationTracker
```

Comment:
O don't see much reason to extract this variable (and name seems to be not relevant).